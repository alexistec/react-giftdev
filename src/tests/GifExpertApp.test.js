import { shallow } from "enzyme"
import GifExpertApp from "../GifExpertApp"

describe('Test in <GifExpertApp/>', () => {
    test('should render correctly', () => {
        const wrapper = shallow( <GifExpertApp/> );
        expect( wrapper ).toMatchSnapshot();

    })
    
})
