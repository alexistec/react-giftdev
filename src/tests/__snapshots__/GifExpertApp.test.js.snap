// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in <GifExpertApp/> should render correctly 1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol>
    <GifGrid
      category="One Punch"
      key="One Punch"
    />
  </ol>
</Fragment>
`;
